### PR TITLE
Custom Scalar Specification URLs

### PR DESCRIPTION
**Pull request by ghostdogpr**
Part of #1159

### CODE REVIEW

File: core/src/main/scala/caliban/schema/Schema.scala [hunk 1]
```diff
@@ -239,18 +246,18 @@ trait GenericSchema[R] extends SchemaDerivation[R] with TemporalSchema {
...
+  implicit val booleanSchema: Schema[Any, Boolean]       = scalarSchema("Boolean", None, None, BooleanValue.apply)
+  implicit val stringSchema: Schema[Any, String]         = scalarSchema("String", None, None, StringValue.apply)
+  implicit val uuidSchema: Schema[Any, UUID]             = scalarSchema("ID", None, None, uuid => StringValue(uuid.toString))
+  implicit val shortSchema: Schema[Any, Short]           = scalarSchema("Short", None, None, IntValue(_))
+  implicit val intSchema: Schema[Any, Int]               = scalarSchema("Int", None, None, IntValue(_))
+  implicit val longSchema: Schema[Any, Long]             = scalarSchema("Long", None, None, IntValue(_))
+  implicit val bigIntSchema: Schema[Any, BigInt]         = scalarSchema("BigInt", None, None, IntValue(_))
+  implicit val doubleSchema: Schema[Any, Double]         = scalarSchema("Float", None, None, FloatValue(_))
+  implicit val floatSchema: Schema[Any, Float]           = scalarSchema("Float", None, None, FloatValue(_))
+  implicit val bigDecimalSchema: Schema[Any, BigDecimal] = scalarSchema("BigDecimal", None, None, FloatValue(_))
```
Comment (paulpdaniels) [inline core/src/main/scala/caliban/schema/Schema.scala:259]: I wonder if we can go ahead and provide urls for any of these. `UUID` for instance does have a specification I believe.

**Review by paulpdaniels — APPROVED** [review]

File: core/src/main/scala/caliban/schema/Schema.scala (hunk 1 above)
Comment (ghostdogpr) [reply core/src/main/scala/caliban/schema/Schema.scala:259]: You mean the official UUID spec, or something specific to GraphQL?

Comment (ghostdogpr) [reply core/src/main/scala/caliban/schema/Schema.scala:259]: Ah, the spec links to `https://tools.ietf.org/html/rfc4122`

Comment (ghostdogpr) [reply core/src/main/scala/caliban/schema/Schema.scala:259]: But we use `ID` for that, which is a standard scalar, and they say that standard scalars shouldn't have that URL.

File: core/src/main/scala/caliban/Rendering.scala [hunk 2]
```diff
@@ -91,6 +94,11 @@ object Rendering {
     case Some(value)            => if (value.contains("\n")) s"""\"\"\"$value\"\"\" """ else s""""$value" """
   }
 
+  private def renderSpecifiedBy(specifiedBy: Option[String]): String = specifiedBy match {
```
Comment (frekw) [inline core/src/main/scala/caliban/Rendering.scala:97]: nit/question: do we prefer to `fold` or `match` in these cases?

Comment (ghostdogpr) [reply core/src/main/scala/caliban/Rendering.scala]: Good point, will use fold. Done.

File: core/src/main/scala/caliban/schema/Schema.scala (hunk 1 above)
Comment (frekw) [reply core/src/main/scala/caliban/schema/Schema.scala:259]: But maybe for the types in `TemporalSchema` could link to the relevant RFCs?

**Pull request merged**